docs/src/modules/java/pages/components.adoc

```diff
@@ -0,0 +1,28 @@
+= Akka SDK Components
+
+include::ROOT:partial$include.adoc[]
+
+== Endpoints
+
+image:ROOT:endpoint.png[float=left, width=30] xref:http-endpoints.adoc[HTTP endpoints] and xref:grpc-endpoints.adoc[gRPC Endpoints]
+
+== Entities
```

Comment:
if we are not explaining the components here I think it would look better to remove the heading all together